docs/docs/guide/debugging_tables/tables.tsx

```diff
@@ -0,0 +1,262 @@
+            <td style={{ textAlign: 'center' }}>✅ ¹</td>
+            <td style={{ textAlign: 'center' }}>✅ ¹</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+          <tr>
+            <td rowSpan={2}>
+              <a href="#chrome-devtools">Chrome DevTools</a>
+            </td>
+            <td style={{ textAlign: 'center' }}>Android</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+          </tr>
+          <tr>
+            <td style={{ textAlign: 'center' }}>iOS</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+          <tr>
+            <td rowSpan={2}>
+              <a href="#flipper-hermes-debugger">Flipper (Hermes debugger)</a>
+            </td>
+            <td style={{ textAlign: 'center' }}>Android</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+          </tr>
+          <tr>
+            <td style={{ textAlign: 'center' }}>iOS</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+          <tr>
+            <td rowSpan={2}>
+              <a href="#safari-devtools">Safari DevTools</a>
+            </td>
+            <td style={{ textAlign: 'center' }}>Android</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+          <tr>
+            <td style={{ textAlign: 'center' }}>iOS</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+          <tr>
+            <td rowSpan={2}>
+              <a href="#react-developer-tools">React Developer Tools</a>
+            </td>
+            <td style={{ textAlign: 'center' }}>Android</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+          </tr>
+          <tr>
+            <td style={{ textAlign: 'center' }}>iOS</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+        </tbody>
+      </table>
+      ¹ - Works, but uses web implementations of function and runs everything on
+      the JS thread.
+      <br />² - Works, but only the JS context can be debugged.
```

Comment:
I feel like we should add a section where we explain the difference between debugging JS runtime (app bundle) vs. UI runtime (worklets).